source/localizable/tutorial/service.md

```diff
@@ -308,3 +308,35 @@ export default Ember.Object.extend({
 We should now see some end to end maps functionality show up on our front page!
 
 ![super rentals homepage with maps](../../images/service/style-super-rentals-maps.png)
+
+### Stubbing Services in Acceptance Tests
+
+Finally, we want to update our acceptance tests to account for our new service.
+While it would be great to verify that a map is displaying, we don't want hammer the Google Maps API every time we run our acceptance test, 
```

Comment:
This sentence is fairly long.  Let's break it at:

our acceptance test _period_  And then remove _so_ below and start a new sentence